forms/common-themes/src/main/resources/theme/base/admin/resources/partials/federated-generic.html

```diff
@@ -5,7 +5,7 @@
         <li data-ng-show="create">Add User Federation Provider</li>
     </ol>
 
-    <h1 data-ng-hide="create"><strong>{{instance.providerName|capitalize}} User Federation Provider</strong> {{instance.displayName|capitalize}}</h1>
+    <h1 data-ng-hide="create"><strong>{{instance.providerName|capitalize}} User Federation Provider</strong></h1>

```

Comment:
Remove 'User Federation Provider'. For clients it's just clientId, so should be consistent.